gateway/api/views/jobs.py

```diff
@@ -76,24 +107,36 @@ def retrieve(self, request, pk=None):  # pylint: disable=unused-argument
         tracer = trace.get_tracer("gateway.tracer")
         ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
         with tracer.start_as_current_span("gateway.job.retrieve", context=ctx):
-            job = Job.objects.filter(pk=pk).first()
+
+            author = self.request.user
+            job = self.jobs_repository.get_job_by_id(pk)
             if job is None:
+                logger.info("Job [%s] nor found", pk)
```

Comment:
I think we can remove this logger message, we are just logging it in the method if I remember correctly 👍